src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
```

Comment:
Please ensure that the property 'spring.application.name' is correctly set to the name of your application.